crates/infra/utils/src/codegen/formatting.rs

```diff
@@ -48,7 +49,7 @@ fn run_formatter(file_path: &Path, contents: &str) -> Result<String> {
     return match get_extension(file_path) {
         "js" | "json" | "ts" => run_prettier(file_path, contents),
         "rs" => run_rustfmt(contents),
-        "html" | "md" | "yml" => {
+        "html" | "md" | "mmd" | "txt" | "yml" => {
```

Comment:
nit: there are no formatters (in CI) for `mmd` or `txt`. I suggest adding it to the below group, along with `ebnf`.